src/frontend/komponenter/Fagsak/FagsakContainer.tsx

```diff
@@ -59,51 +60,77 @@ const FagsakContainer: React.FunctionComponent = () => {
             switch (brukerRessurs.status) {
                 case RessursStatus.SUKSESS:
                     return (
-                        <>
-                            <Personlinje
-                                bruker={brukerRessurs.data}
-                                minimalFagsak={minimalFagsak.data}
-                            />
-                            <HovedInnhold>
-                                <Routes>
-                                    <Route
-                                        path="/saksoversikt"
-                                        element={
+                        <HovedInnhold>
+                            <Routes>
+                                <Route
+                                    path="/saksoversikt"
+                                    element={
+                                        <>
+                                            <Personlinje
+                                                bruker={brukerRessurs.data}
+                                                minimalFagsak={minimalFagsak.data}
```

Comment:
Dette blir smudere når jeg har laget en egen context for bruker og fagsak som ikke holder på en ressurs, slik som jeg har gjort med behandling. Da slipper man å sende med disse som props

Reply:
Nå må vi gjenta Personlinje for hver url i fagsakContaineren, men synes det er bedre siden da kan vi ha mer kontroll over contextene, siden vi kan flytte behandlingscontexten innover